### PR TITLE
feat: go to helplines page if necessary

### PR DESCRIPTION
**Pull request by jthistle**
https://trello.com/c/ffCEpxbw/120-conditional-link-to-helplines-missing

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/quiz/how-are-you-feeling**](https://my-best-life-dcacyfgfy-neontribe.vercel.app/quiz/how-are-you-feeling) |
|:-:|
|<a href="https://my-best-life-dcacyfgfy-neontribe.vercel.app/quiz/how-are-you-feeling">Screenshot of /quiz/how-are-you-feeling</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/FQcjio2I3sykknFzZfhcbkeunm4_.png">(view full size)</a> |







Commit 6ab16e224a2966743319c1b9919f4f2dd738eca8 (https://my-best-life-dcacyfgfy-neontribe.vercel.app).

File: pages/quiz/how-are-you-feeling.tsx [hunk 1]
```diff
@@ -248,10 +256,9 @@ export const HowAreYouFeelingPage = (): JSX.Element => {
           />
         </QuestionSection>
 
-        <Link href="/quiz/what-are-your-interests" passHref>
```
Comment (Nikomus) [inline pages/quiz/how-are-you-feeling.tsx:251]: We could keep this as a link and just choose the other URL

**Review by Nikomus — APPROVED** [review]
Optional change, but looks to do the right thing ☎️

File: pages/quiz/how-are-you-feeling.tsx (hunk 1 above)
Comment (jthistle) [reply pages/quiz/how-are-you-feeling.tsx:251]: This is true

**Pull request merged**